### DISCUSSION TITLE
Tags to build individualized schedules

### DISCUSSION THREAD

**Discussion by Claudenw** [EventFahrplan/EventFahrplan#613] [Ideas]
Greetings,  I am the Chair of Community over [Code Europe](https://www.codeeurope.pl), 2024  We are trying to build a conference where we tag the talks with multiple track IDs.  The idea is that many of the best talks address issues across tracks.  So a big data talk may be of interest to people in the Hadoop track, or similar cross track interest.  

What I want to do is tag our talks with virtual track IDs so that attendees can find all the talks that address big data regardless of what official track they are in.  

if EventFahrplan had the ability to store multiple tags (hashtags?) for a track and allow searching the schedule for talks that contain tags (all or any) so that attendees can build the best path through all the talks.

Is this of any interest to anyone else?

**Comment by johnjohndoe** [comment]
Hello @Claudenw, thanks for your interest in this project and for opening the discussion here.

Currently, organizers associate a **single track** for a session to communicate its **main topic**. That is primarily limited by the software used in the backend (Frab, Pretalx, Wafer) respectively the data format being used. In this respect, the app simply displays the track name text which comes with each session.

I realized some time ago that session tagging would be a valuable feature. There is a [draft](https://github.com/EventFahrplan/EventFahrplan/issues/12) explaining the concept. I was thinking more about **giving users the ability to tag sessions** themselves. I think this would open up many more possibilities than one could come up with for them.

Once someone has time to work on this feature users would also be able to search and filter by tags to compose very customized schedules for themselves.

# Related
- https://github.com/voc/schedule/issues/131
